_includes/neurips/2019/about.neurips.2019.html

```diff
@@ -126,6 +126,9 @@ <h3>Bronze</h3>
 		  <a target="_blank" href="https://www.intel.com"><img width=250
                     src="./assets/img/icml/2019/sponsors/Bronze-Intel.png"></a>
                   </br>
+		  <a target="_blank" href="https://www.twosigma.com"><img width=250
+                    src="./assets/img/icml/2019/sponsors/TwoSigma.png"></a>
```

Comment:
The file source reference is incorrect and the logo is not displaying. Should be "./assets/img/icml/2019/sponsor/TwoSigma.png"

Reply:
Done.
I don't understand what happened to the apple logo I did upload the new !!!
I didn't notice the 2 sigma but now is actually fine